src/com/studentmanagementsystem/data/list/ArrayListStudentDAO.java

```diff
@@ -0,0 +1,32 @@
+package com.studentmanagementsystem.data.list;
+
+import com.studentmanagementsystem.logic.Student;
+import com.studentmanagementsystem.logic.StudentDAO;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+
+public class ArrayListStudentDAO implements StudentDAO {
+    ArrayList<Student> students;
```

Comment:
Why default access modifier? This property is only used inside the DAO, use private!